examples/common_vmseries/example.tfvars

```diff
@@ -217,3 +217,102 @@ vmseries = {
     ]
   }
 }
+
+# --- TEST INFRASTRUCTURE PART --- #
+
+test_environments = {
+  "east-testenv" = {
+    vnets = {
+      "spoke_east" = {
```

Comment:
If the general use is to use a single VNet for a test environment maybe we can convert vnets into a single test vnet map? And eventually the module vnets varilable as well.

Reply:
As discussed, we're going to leave everything as map(object).